pkg/oam/labels.go

```diff
@@ -99,4 +99,10 @@ const (
 
 	// AnnotationKubeVelaVersion is used to record current KubeVela version
 	AnnotationKubeVelaVersion = "oam.dev/kubevela-version"
+
+	// AnnotationFilterAnnotationKeys is used to filter annotations passed to workload and trait
```

Comment:
Indicate that keys are joint by `,` in the annotation.

Reply:
done